src/test/Makefile

```
@@ -122,7 +122,7 @@ cms_unittest_SRC := \
 		$(USER_DIR)/cms/cms.c \
 		$(USER_DIR)/cms/cms_menu_saveexit.c \
 		$(USER_DIR)/common/typeconversion.c \
-		$(USER_DIR)/drivers/display.c
+		$(USER_DIR)/drivers/display.c \
```

No need for continuation

```suggestion
		$(USER_DIR)/drivers/display.c
```